### PR TITLE
Add theming/configurability support to website

### PR DESCRIPTION
**Pull request by ronlaniado**
You can now easily theme the website via the `globals.css` and `constants.js` files.

### CODE REVIEW

File: ThemeConfig.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* :root {
```
Comment (CodingKoopa) [inline ThemeConfig.css]: What is this file for? It looks like `globals.css`, but I don't see it referenced anywhere.

File: components/NavBar.js [hunk 2]
```diff
@@ -1,53 +1,50 @@
...
-import constants from "../public/constants.js"
+import constants from "../public/constants.js";
 
 export default function NavBar() {
+  const [active, setActive] = useState(0);
 
-    const [active, setActive] = useState(0);
+  function handleClick(i) {
+    setActive(i);
+  }
```
Comment (CodingKoopa) [inline components/NavBar.js]: Consider inlining this since it's just one function call.

File: package-lock.json [hunk 3]
```diff
@@ -1,5340 +1,7 @@
 {
```
Comment (CodingKoopa) [inline package-lock.json]: I think that having a lockfile for both NPM and Yarn is dangerous. Conceivably, the two can get out of date, leading to the developer getting different package versions depending on whether they use NPM or Yarn.

Consider deleting one of the lockfiles and enforcing one of the two package managers in the readme.

File: public/constants.js [hunk 4]
```diff
@@ -1,3 +1,40 @@
+import heroImage from "../public/assets/heroImage.webp";
```
Comment (CodingKoopa) [inline public/constants.js:1]: The location of this file struck me as odd at first. Is this code going to be executed by the client, or is it just consumed during the build process? If the latter, I am not sure if it belongs in the `public` directory.

**Review by CodingKoopa — CHANGES_REQUESTED** [review]
Thanks! I left some comments about things I have questions about.

File: ThemeConfig.css (hunk 1 above)
Comment (ronlaniado) [reply ThemeConfig.css]: Oops that was supposed to be deleted in another commit!

File: public/constants.js (hunk 4 above)
Comment (ronlaniado) [reply public/constants.js:1]: What do you mean by consumed during the build process?

File: package-lock.json (hunk 3 above)
Comment (ronlaniado) [reply package-lock.json]: Yarn lives on

File: public/constants.js (hunk 4 above)
Comment (CodingKoopa) [reply public/constants.js:1]: By that, I mean that it is not directly served to the client.

Comment (ronlaniado) [reply public/constants.js:1]: Oh yeah this _is_ served to the client.

**Pull request merged**